### PR TITLE
let enikshay reports through

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

File: corehq/apps/domain/middleware.py [hunk 1]
```diff
@@ -72,6 +72,11 @@ def process_view(self, request, view_func, view_args, view_kwargs):
             if getattr(view_func, 'domain_migration_handled', False):
                 return None
             if DATA_MIGRATION.enabled(request.domain):
+                # hack for enikshay
+                url_parts = request.path.split('/')
+                if len(url_parts) > 3 and url_parts[3] in ['reports', 'toggles.js']:
```
Comment (mkangia) [inline corehq/apps/domain/middleware.py]: why do we need toggle.js?

**Review by gcapalbo — APPROVED** [review]
seems ok to do for data checkout now, but i think we should revert this once we no longer need it

File: corehq/apps/domain/middleware.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/domain/middleware.py]: This gets loaded with reports and triggers the migration in progress page

**Comment by nickpell** [comment]
@gcapalbo Agree, will revert afterwards

File: corehq/apps/domain/middleware.py (hunk 1 above)
Comment (mkangia) [inline corehq/apps/domain/middleware.py]: okay cool.

**Review by mkangia — APPROVED** [review]

**Pull request merged**